### DISCUSSION TITLE
Designing Circuits Using SKiDL and ChatGPT/GPT4

### DISCUSSION THREAD

**Discussion by devbisme** [xesscorp/skidl#181] [Show and tell]
I wrote a [blog post](https://devbisme.github.io/skidl/skidl-meets-chatgpt-2023-03-23.html) about a Twitter discussion and experiment that we ran to design circuitry by having ChatGPT/GPT4 output SKiDL code.

https://devbisme.github.io/skidl/skidl-meets-chatgpt-2023-03-23.html

**Comment by szolotykh** [comment]
Hi. I am working of proof of concept of text base schematic generator. Yesterday, I found SKiDL and really excited about this project. I think, it is better way to represent schematic for LLM than for example Kicad schematic files.

**Reply by devbisme** [reply]
Keep us posted on your progress!
